src/frontend/js/components/UserLogin/index.tsx

```diff
@@ -73,11 +73,12 @@ export const UserLogin = ({ loginUrl, logoutUrl, signupUrl }: UserLoginProps) =>
 
   return (
     <div className="user-login">
-      {user === undefined ? (
+      {user === undefined && (
```

Comment:
It seems to me that by writing it like this we're losing the meaning that only one of those three blocks should be shown at any time. If we add a condition to one of these it's easier to create a bug.

Maybe it's not that important. Not sure which is better, I wish javascript `if` statements were expressions :(